node_modules/game-iverse/platformer/levels.js

```diff
@@ -0,0 +1,178 @@
+.......#=.o..#............#...................................###.#.###.#.###.#.###.#.###.#.###.#.#####.......
+.......#.....#..................................................#.#...#.#...#.#...#.#...#.#...#.#.....#.......
+.......+..o..+............o..................................####.#####.#####.#####.#####.#####.#######.......
+..............................................................................................................
+..........o..............###..............................##..................................................
+..............................................................................................................
+..............................................................................................................
+......................................................##......................................................
+...................###.........###............................................................................
+..............................................................................................................
+..........................o.....................................................#......#......................
+..........................................................##.....##...........................................
+.............###.........###.........###.................................#..................#.................
+..............................................................................................................
+.................................................................||...........................................
+..###########.................................................................................................
+..#.........#.o.#########.o.#########.o.##................................................#...................
+..#.........#...#.......#...#.......#...#.................||..................#.....#.........................
+..#..@......#####...o...#####...o...#####.....................................................................
+..#######.....................................#####.......##.....##.....###...................................
+........#=..................=................=#...#.....................###...................................
+........#######################################...#+++++++++++++++++++++###+++++++++++++++++++++++++++++++++++
+..................................................############################################################
+..............................................................................................................
+`];
+
+if (typeof module != "undefined" && module.exports && (typeof window == "undefined" || window.exports != exports))
+  module.exports = GAME_LEVELS;
+if (typeof global != "undefined" && !global.GAME_LEVELS)
+  global.GAME_LEVELS = GAME_LEVELS;
```

Comment:
You should move this to the `platformer/READEME.md` and your test repository, I would like to try it out myself.
After I publish the new version we'll make a webpage to display our games. its just an NPM package shouldn't have a html doc to run. It should just be modules to be exported and imported. 

However in the npm package we shouldn't have a full game.